docker/server/Dockerfile

```diff
@@ -1,22 +1,24 @@
 # Stage 1: Download all dependencies
 FROM ubuntu:20.04 AS dependencies
 
-RUN apt-get update && apt-get install -y --no-install-recommends cmake
+# Stage 1.1: Setup kitware repository
+## According to: https://apt.kitware.com/
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends gpg wget ca-certificates && \
```

Comment:
set `DEBIAN_FRONTEND=noninteractive` globally, so you don't need to pass on every command